RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate AttachmentView to Typescript

## Proposed changes


## Issue(s)	


## How to test or reproduce


- You should be able to view files uploaded to your rooms
- Furthermore, check if the Attachment View

## Screenshots

## Types of changes




## Checklist



## Further comments
